checks/all_checks.go

```diff
@@ -16,18 +16,43 @@
 package checks
 
 import (
+	"os"
+
 	"github.com/ossf/scorecard/v4/checker"
 )
 
-// AllChecks is the list of all security checks that will be run.
-var AllChecks = checker.CheckNameToFnMap{}
+// allChecks is the list of all registered security checks
+var allChecks = checker.CheckNameToFnMap{}
+
+func getAll(overrideExperimental bool) checker.CheckNameToFnMap {
+	// need to make a copy or caller could mutate original map
+	possibleChecks := checker.CheckNameToFnMap{}
+	for k, v := range allChecks {
+		possibleChecks[k] = v
+	}
+
+	if overrideExperimental {
+		return possibleChecks
+	}
+
+	// TODO: remove this check when v6 is released
+	if _, v6 := os.LookupEnv("SCORECARD_V6"); !v6 {
```

Comment:
This PR might be a good chance for us to move to `SCORECARD_EXPERIMENTAL` instead of V6. If its not a lot of work, would you mind changing that? I don't think it should break any users.

Reply:
Sure